common/libexec/wwinit/10-database.init

```diff
@@ -90,45 +115,35 @@ fi
 
 if [ -z "$DBVERS" ]; then
     wwprint "Creating database schema"
-    if echo "$SCHEMA" | mysql "$DBNAME"; then
+    if echo "$SCHEMA" | mysql $DBOPTIONS "$DBNAME"; then
```

Comment:
My bad, I forgot we had the entire schema in a variable scalar. It really should be in a file so we can `cat schema.txt | mysql ...`. Using `echo` can be tricky if there are any other quotes, escapes or anything else in the string, as `echo` and/or the shell can parse them.

This is fine considering the simple schema, sorry for the alarm.